autonomy/cli/mint.py

```diff
@@ -62,10 +62,16 @@
 @click.group("mint")
 @pass_ctx
 @chain_selection_flag_()
-def mint(ctx: Context, chain_type: str) -> None:
+@click.option(
+    "--skip-hash-check",
```

Comment:
Seems a bit out place please provide any better name for this flag if you have any